metadata.json

```diff
@@ -7,38 +7,68 @@
   "source": "https://github.com/MiamiOH/puppet-pam_access.git",
   "project_page": "https://github.com/MiamiOH/puppet-pam_access",
   "issues_url": "https://github.com/MiamiOH/puppet-pam_access/issues",
-  "tags": ["pam", "access"],
-  "description": "The pam_access module allows access control based on username, groupname, network address, and process name.  This module allows Puppet to manage entries in the access.conf file.",
   "dependencies": [
-    {"name": "puppetlabs/stdlib", "version_requirement": ">= 4.1.0 < 7.0.0"},
-    {"name": "herculesteam/augeasproviders_pam", "version_requirement": ">= 2.1.0 < 3.0.0"}
+    {
+      "name": "puppetlabs/stdlib",
+      "version_requirement": ">= 4.1.0 < 6.0.0"
```

Comment:
this has changed from 7.0.0 to 6.0.0. Did we want this?

Reply:
yes, One step a time